google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -78,12 +95,41 @@ def __init__(
                 "Arg instance_connection_string must be in "
                 + "format: project:region:instance."
             )
+
         self._loop = loop
         self._auth_init()
         self._priv_key, self._pub_key = generate_keys()
+        self._mutex = threading.Lock()
+
+        async def create_client_session():
+            return aiohttp.ClientSession()
+
+        self._client_session = asyncio.run_coroutine_threadsafe(
+            create_client_session(), loop=self._loop
+        ).result()
+
+        print("instance data")
```

Comment:
Should this print statement be here? 

Might be worth considering using `logging` instead  so that we can leave these and use them to debug. Ask Chris how to determine which logger should be used.

Reply:
same for other print statements